osquery/sql/sqlite_util.cpp

```diff
@@ -223,6 +223,25 @@ class StringEscaperVisitor : public boost::static_visitor<> {
   }
 };
 
+class SizeVisitor : public boost::static_visitor<> {
+ public:
+  template <typename T>
+  void operator()(T& t) const {
+    size += sizeof(t);
```

Comment:
size += sizeof(t); or size = sizeof(t);?

If type is string m I believe there should be an overloaded operator for that sth like ?
void operator()(const std::string& t) {
  size = t.size();
}